lib/sort_by_length.rb

```diff
@@ -1,7 +1,15 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(n)
 def sort_by_length(my_sentence)
```

Comment:
This is a selection, sort works, but requires an external array.  Could you instead do it in-place?